docs/license-management.md

```diff
@@ -0,0 +1,13 @@
+## Activate a Xen Orchestra license
```

Comment:
Use a first level title in your page, with a short intro, eg:

# License management

Lorem Ipsum dolor, why license management, etc.

## Activate a Xen Orchestra license